src/resources/views/auth/account/sidemenu.blade.php

```diff
@@ -1,4 +1,10 @@
 <div class="box">
+    <div class="box-body box-profile">
+
+	    <img class="profile-user-img img-responsive img-circle" src="{{ backpack_avatar_url(Auth::user()) }}">
```

Comment:
And if that `Auth::user()` returns `null`?

Reply:
If you look at sidebar.blade.php (which is where i copied the line from) it is exactly the same so look at @tabacitu for that answer.

Reply:
If it's `null` we'll get a PHP error; I guess seeing we're meant to be logged in here it won't ever be `null`.